### ISSUE TITLE
Failing unit tests

### ISSUE THREAD

**Issue by VinQbator** [segmentio/action-destinations#327]
Cloned the repo and followed setup instructions.

`yarn --ignore-optional` did complain about ssh://git@github.com/segmentio/control-plane-service-js-client.git not being available. Was not able to get it to work with removing the deps as recommended in https://github.com/segmentio/action-destinations/issues/169

When running `yarn test`, I'm getting:
```

...
@segment/action-destinations: FAIL src/destinations/google-analytics-4/__tests__/purchase.test.ts
@segment/action-destinations:   ● Test suite failed to run
@segment/action-destinations:     ../core/src/schema-validation.ts:1:35 - error TS2307: Cannot find module '@segment/ajv-human-errors' or its corresponding type declarations.
@segment/action-destinations:     1 import { AggregateAjvError } from '@segment/ajv-human-errors'
@segment/action-destinations:                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
@segment/action-destinations: FAIL src/destinations/personas-messaging-sendgrid/__tests__/send-email.test.ts
@segment/action-destinations:   ● Test suite failed to run
@segment/action-destinations:     ../core/src/schema-validation.ts:1:35 - error TS2307: Cannot find module '@segment/ajv-human-errors' or its corresponding type declarations.
@segment/action-destinations:     1 import { AggregateAjvError } from '@segment/ajv-human-errors'
@segment/action-destinations:                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~
@segment/action-destinations: FAIL src/destinations/braze/__tests__/braze.test.ts
...
```

Also tried installing `ajv` as recommended during yarn init step `warning " > @segment/ajv-human-errors@2.0.1" has unmet peer dependency "ajv@^8.0.0".` with `npm install ajv@^8.0.0`, but that did not seem to help.

**Comment by mshwery** [comment]
It's normal to see complaints with `--ignore-optional`, but they should be warnings only. Did it successfully install or were there errors that exited?

You shouldn't need to install ajv separately, either! The warning is expected unfortunately, and despite having ajv installed as a devDep already, the warning still appears. 

Did you run `yarn bootstrap` after `yarn install --ignore-optional` ?

**Comment by bgila** [comment]
i ran into this issue too -- but running `yarn build` before `yarn test` resolved it for me.

**Comment by VinQbator** [comment]
> It's normal to see complaints with `--ignore-optional`, but they should be warnings only. Did it successfully install or were there errors that exited?

```
  yarn --ignore-optional
yarn install v1.16.0
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
error https://registry.yarnpkg.com/@segment/remote-service-client/-/remote-service-client-2.4.1.tgz: Request failed "404 Not Found"
error Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads ssh://git@github.com/segmentio/control-plane-service-js-client.git
Directory: /Users/***/action-destinations
Output:
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
[4/5] 🔗  Linking dependencies...
warning " > @segment/ajv-human-errors@2.0.1" has unmet peer dependency "ajv@^8.0.0".
warning "workspace-aggregator-933af8da-1696-45d3-ae98-0276de1e1e2e > @segment/actions-cli-internal > jscodeshift@0.13.0" has unmet peer dependency "@babel/preset-env@^7.1.6".
warning "workspace-aggregator-933af8da-1696-45d3-ae98-0276de1e1e2e > @segment/actions-cli-internal > jscodeshift-add-imports@1.0.10" has incorrect peer dependency "jscodeshift@^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0".
warning "workspace-aggregator-933af8da-1696-45d3-ae98-0276de1e1e2e > @segment/actions-cli-internal > jscodeshift-add-imports > jscodeshift-find-imports@2.0.4" has incorrect peer dependency "jscodeshift@^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0".
[5/5] 🔨  Building fresh packages...
$ husky install
husky - Git hooks installed
✨  Done in 12.97s.
```

> Did you run `yarn bootstrap` after `yarn install --ignore-optional` ?

```
  yarn bootstrap
yarn run v1.16.0
$ lerna bootstrap
lerna notice cli v4.0.0
lerna info versioning independent
lerna info bootstrap root only
[1/4] 🔍  Resolving packages...
success Already up-to-date.
$ husky install
husky - Git hooks installed
✨  Done in 2.23s.
```

**Comment by VinQbator** [comment]
> i ran into this issue too -- but running `yarn build` before `yarn test` resolved it for me.

Tests did pass after running `yarn build` 🥳 

Probably should update readme, so that JS noobs could work on it too :)

**Comment by mericsson** [comment]
> > i ran into this issue too -- but running `yarn build` before `yarn test` resolved it for me.
> 
> Tests did pass after running `yarn build` 🥳
> 
> Probably should update readme, so that JS noobs could work on it too :)

@VinQbator thanks good feedback! fyi @gpsamson

**Comment by mshwery** [comment]
Ah this is interesting and I believe unexpected! Thanks for reporting. :)

**Comment by gpsamson** [comment]
This has been fixed with #340. There were some missing `ts-jest` configs that were causing failed imports. I'll close this issue in a few days in case folks are still running into this.

To help catch any workflow regressions, we have a CI workflow that runs the setup commands outlined in the README and ensure they don't fail: https://github.com/segmentio/action-destinations/blob/main/.github/workflows/ext.yml
